CHANGES.txt

```diff
@@ -33,3 +33,10 @@ Bugfix
   MIMEAccept to behave more like the old version. See
   https://github.com/Pylons/webob/pull/356
 
+- ``acceptparse.AcceptValidHeader``, ``acceptparse.AcceptInvalidHeader``, and
+  ``acceptparse.AcceptNoHeader`` will now always ignore offers that do not
+  match the required media type grammar when calling ``.acceptable_offers()``.
+  Previous versions raised a ``ValueError`` for invalid offers in
+  ``AcceptValidHeader`` and returned them as acceptable in the others.
+  See https://github.com/Pylons/webob/pull/371
```

Comment:
Is this URL intended to be the URL of this PR (372 instead of 371)?

Reply:
fixed, thanks